requirements.txt

```diff
@@ -11,6 +11,7 @@ django-cas-ng>=3.6
 django-cors-headers
 django-debug-toolbar
 django-prometheus
+django-redis-sessions
```

Comment:
Session caching lib that shares Redis with Cacheops.

Reply:
You need to include this also in the setup file within the extra `cache` property
https://github.com/eHealthAfrica/aether-django-sdk-library/blob/master/setup.py#L69-L71

Also it would be appreciated if you add the entry in the README
https://github.com/eHealthAfrica/aether-django-sdk-library/blob/master/README.md#requirements